tests/pausing_test.go

```diff
@@ -209,11 +209,11 @@ var _ = Describe("[rfe_id:3064][crit:medium][vendor:cnv-qe@redhat.com][level:com
 
 		When("paused via API", func() {
 			It("[test_id:4598]should signal paused state with condition", func() {
-				err = virtClient.VirtualMachineInstance(vm.Namespace).Pause(context.Background(), vm.Name, &v1.PauseOptions{})
+				err = virtClient.VirtualMachineInstance(testsuite.GetTestNamespace(vm)).Pause(context.Background(), vm.Name, &v1.PauseOptions{})
```

Comment:
How come this happens?  The `vm, err = virtClient.VirtualMachine(testsuite.GetTestNamespace(vm)).Create(context.Background(), vm, v12.CreateOptions{})` should populate the namespace.

Reply:
You are right :) 
I removed this commit.